package.json

```diff
@@ -99,7 +99,7 @@
     "@typescript-eslint/parser": "4.15.2",
     "@woocommerce/api": "0.2.0",
     "@woocommerce/components": "12.0.0",
-    "@woocommerce/csv-export": "1.7.0",
+    "@woocommerce/csv-export": "1.8.0",
```

Comment:
May change once the package is [published](https://github.com/woocommerce/woocommerce/blob/trunk/tools/package-release/README.md).

Reply:
Confirming that 1.8.0 it is:  
https://www.npmjs.com/package/@woocommerce/csv-export